LovelyCrane/SpriteKit/SpriteScene.swift

```diff
@@ -19,16 +19,20 @@ final class SpriteScene: SKScene {
 
     override func didMove(to view: SKView) {
         physicsBody = SKPhysicsBody(edgeLoopFrom: frame)
-        backgroundColor = .red
         motionManager?.setCoreMotionManager()
-        createCrane()
+        backgroundColor = .backGround
+        for _ in 0...40 {
+            createCrane()
+        }
     }
     
     private func createCrane() {
-        let crane = SKSpriteNode(imageNamed: Assets.inbox)
-        crane.physicsBody = SKPhysicsBody(texture: crane.texture!, size: crane.texture!.size())
-        crane.position = CGPoint(x: CGFloat.random(in: 0...size.width), y: size.height * 0.9)
-        addChild(crane)
+        if let randomCrane = Assets.crans.randomElement()?.rawValue {
+            let crane = SKSpriteNode(imageNamed: randomCrane)
+            crane.physicsBody = SKPhysicsBody(texture: crane.texture!, size: crane.texture!.size())
+            crane.position = CGPoint(x: CGFloat.random(in: 0...size.width), y: size.height * 0.9)
+            addChild(crane)
+        }
```

Comment:
```suggestion
        guard let randomCrane = Assets.crans.randomElement()?.rawValue else { return }
        let crane = SKSpriteNode(imageNamed: randomCrane)
        crane.physicsBody = SKPhysicsBody(texture: crane.texture!, size: crane.texture!.size())
        crane.position = CGPoint(x: CGFloat.random(in: 0...size.width), y: size.height * 0.9)
        addChild(crane)
```

guard let을 쓰면 인덴트를 줄일 수 있어보여요 ~ !!

Reply:
감사합니다! 반영해두도록 하겠습니다.